hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/consumer/receiver/kafka/KafkaSingleThreadedMessageReceiver.java

```diff
@@ -110,7 +110,7 @@ public KafkaSingleThreadedMessageReceiver(KafkaConsumer<byte[], byte[]> consumer
         } catch (InterruptException ex ) {
             // Despite that Thread.currentThread().interrupt() is called in InterruptException's constructor
             // Thread.currentThread().isInterrupted() somehow returns false so we reset it.
-            logger.info("Kafka consumer thread interrupted", ex);
+//            logger.info("Kafka consumer thread interrupted", ex);
```

Comment:
Why you have commented this log?

Reply:
It produces a lot of log messages. Not useful during debug.